cmd/installer/cli/install.go

```diff
@@ -90,7 +91,7 @@ type installConfig struct {
 	isAirgap                bool
 	enableManagerExperience bool
 	licenseBytes            []byte
-	license                 *kotsv1beta1.License
+	license                 licensewrapper.LicenseWrapper
```

Comment:
it seems safer to pass a pointer to the license wrapper around given that we had a pointer to license. that would allow for `license != nil` rather than `license.GetLicenseID() != ""`